doc/profile_handling.md

```diff
@@ -0,0 +1,104 @@
+# Profile Handling
```

Comment:
Please add link to this document also in README.md so it is easier to discover it.

Reply:
Good point.